packages/components/nodes/chains/SqlDatabaseChain/SqlDatabaseChain.ts

```diff
@@ -105,19 +140,49 @@ class SqlDatabaseChain_Chains implements INode {
         const databaseType = nodeData.inputs?.database as DatabaseType
         const model = nodeData.inputs?.model as BaseLanguageModel
         const url = nodeData.inputs?.url
-        const customPrompt = nodeData.inputs?.customPrompt as string
-
-        const chain = await getSQLDBChain(databaseType, url, model, customPrompt)
+        const includesTables = nodeData.inputs?.includesTables
+        const splittedIncludesTables = includesTables == '' ? undefined : includesTables?.split(',')
+        const ignoreTables = nodeData.inputs?.ignoreTables?.split(',')
+        const splittedIgnoreTables = ignoreTables == '' ? undefined : includesTables?.split(',')
```

Comment:
`ignoreTables` is an array after split, so `ignoreTables == ''` is not a valid if else check. Also why `includesTables?.split(',')` is being used instead of `ignoreTables`?

Reply:
ops sorry for typo

Reply:
should it be `ignoreTables?.split(',')` ?

Reply:
yes, copy paste error..